recipe/build.sh

```diff
@@ -4,17 +4,62 @@ set -euxo pipefail
 
 if [[ "${target_platform}" == osx-* ]]; then
   export LDFLAGS="${LDFLAGS} -lz -framework CoreFoundation -Xlinker -undefined -Xlinker dynamic_lookup"
+  # Remove stdlib=libc++; this is the default and errors on C sources.
+  export CXXFLAGS=${CXXFLAGS//-stdlib=libc++}
 else
   export LDFLAGS="${LDFLAGS} -lrt"
 fi
+export CFLAGS="${CFLAGS} -DNDEBUG"
+export CXXFLAGS="${CXXFLAGS} -DNDEBUG"
 
-export LD_LIBRARY_PATH="${LD_LIBRARY_PATH+LD_LIBRARY_PATH:}:$PREFIX/lib"
+export BUILD_FLAGS="--target_cpu_features default --enable_mkl_dnn"
+
+#  - if JAX_RELEASE or JAXLIB_RELEASE are set: version looks like "0.4.16"
+#  - if JAX_NIGHTLY or JAXLIB_NIGHTLY are set: version looks like "0.4.16.dev20230906"
+#  - if none are set: version looks like "0.4.16.dev20230906+ge58560fdc
+export JAXLIB_RELEASE=1
+
+if [[ ${cuda_compiler_version} != "None" ]]; then
+  export HERMETIC_CUDA_COMPUTE_CAPABILITIES=sm_60,sm_70,sm_75,sm_80,sm_86,sm_89,sm_90,compute_90
+  export CUDA_HOME="${BUILD_PREFIX}/targets/x86_64-linux"
```

Comment:
I think CUDA_HOME should be set in the nvcc activation script: https://github.com/AnacondaRecipes/nvcc-feedstock/blob/6155eddca934e13ea2c1d3575efecd73e571cade/recipe/install_nvcc.sh#L5

Reply:
Most of these comments are from things that were inspired from the approved tensorflow PR here: https://github.com/AnacondaRecipes/tensorflow-feedstock/pull/1 If they need to be addressed here, do we need to address them there too?

Reply:
probably? I guess I'm trying to set a standard for how the cuda stuff should be handled.

Not sure whether it's more pragmatic to write up a document for discussion and agreement, or just do it. I'd personally prefer the second, also documenting in perseverance-skills.

Don't want to put work on your desk though so happy to do it if required. Maybe we could open a jira ticket.

I couldn't see anything about CUDA_HOME in the tensorflow feedstock though. Is it needed here to make it work? Maybe a bazel thing.